packages/portal/spec-renderer/src/components/SpecDetails.cy.ts

```diff
@@ -42,6 +42,19 @@ describe('<SpecDetails />', () => {
     cy.getTestId('hide-essentials-styles').should('exist')
   })
 
+  // TODO: evaluate if we need to test this
```

Comment:
@kaiarrowood I don't think we need this test to check for styles anymore, shall I remove it?

Reply:
Removed